src/platform/graphics/webgl/webgl-texture.js

```diff
@@ -451,6 +462,42 @@ class WebglTexture {
                                   this._glPixelType,
                                   mipObject);
                 }
+            } else if (texture._array && typeof mipObject === "object") {
```

Comment:
I think this might need to be adjusted as there were recent changes in the engine to the code around here, please try to integrate main to your branch

Reply:
I have updated to latest main but I haven't seen any major changes that affects this.